requirements.txt

```diff
@@ -5,4 +5,6 @@ pylint
 vcrpy
 contextlib2
 mock
-jsonpickle
\ No newline at end of file
+jsonpickle
```

Comment:
Remove the jsonpickle requirement. This shouldnt be needed anymore as it was removed previously and we are not allowed to use anything besides the standard json lib.